webpack.config.js

```diff
@@ -3,42 +3,18 @@ const path = require('path'),
     CopyWebpackPlugin = require('copy-webpack-plugin'),
```

Comment:
THANK YOU so much for taking a look at this. I haven't looked at this config file like ever.... I'm scared of it!